RELEASE/scripts/autoscend/auto_restore.ash

```diff
@@ -393,6 +393,10 @@ __RestorationOptimization __calculate_objective_values(int hp_goal, int mp_goal,
       restored_amount += numeric_modifier("Bonus Resting HP");
     }
 
+    if (isActuallyEd() && metadata.name != "linen bandages") {
```

Comment:
just for completeness: also cotton bandages and silk bandages, I think

Reply:
fixed.